Userland/Utilities/bt.cpp

```diff
@@ -37,10 +37,14 @@ int main(int argc, char** argv)
 
     while (iterator.has_next()) {
         pid_t tid = iterator.next_path().to_int().value();
-        outln("tid: {}", tid);
+        outln("thread: {}", tid);
+        outln("frames:");
         auto symbols = Symbolication::symbolicate_thread(pid, tid);
+        auto frame_number = symbols.size() - 1;
         for (auto& symbol : symbols) {
-            out("{:p}  ", symbol.address);
+            // Make kernel stack frames stand out.
+            int color = symbol.address < 0xc0000000 ? 35 : 31;
+            out("{:3}: \033[{};1m{:p}\033[0m | ", frame_number, color, symbol.address);
```

Comment:
The `| ` would kinda stand out if the symbol doesn't have a name (the branch immediate below), maybe print it in that branch?
Here's a little test so we can compare:
```
 10: 0xc0369696 | Kernel::FancyName::foo()+69
  9: 0xc03ed1bd |
  8: 0xc03badad | Kernel::FancyName::bar()+42
```
or
```
 10: 0xc0369696 | Kernel::FancyName::foo()+69
  9: 0xc03ed1bd
  8: 0xc03badad | Kernel::FancyName::bar()+42
```

Reply:
:thinking: I guess it doesn't really change a whole lot...

Reply:
Nah, let's keep it like this for easier scriptability (should be easier to `cut -d\| -f2`)